console/src/main/webapp/WEB-INF/i18n/application.properties

```diff
@@ -153,3 +153,5 @@ shortName.label=Short name
 name.label=Name
 address.label=Address
 orgType.label=Organization Type
+preauth.name.label=Identity Provider
+preauth.uid.label=User ID
```

Comment:
I think we are mixing 2 concepts:

* preauthentication: which is the mechanism where the authentication is performed by another component before hitting georchestra
* external identity providers

if we are using the same labels, maybe we should find something more "generic" for both concepts.